proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -253,7 +254,16 @@ message UpdateReviewFlowGracePeriodPolicyRequest {
 
 message UpdateReviewFlowGracePeriodPolicyResponse {}
 
-message GetClusterUsageRequest {}
+message GetClusterUsageRequest {
+  // The timestamp of the beginning time of a date range.
+  google.protobuf.Timestamp start_time = 1;
+  // The timestamp of the ending time of a date range.
+  google.protobuf.Timestamp end_time = 2;
+
+  optional string organization_id = 3;
```

Comment:
Why are these being marked optional?

Reply:
Why were they marked optional though? Was there a reason as opposed to the default concept of optional in proto3?